frontend/packages/kubevirt-plugin/src/components/vms/vm-users.tsx

```diff
@@ -0,0 +1,152 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { watchURL, stopWatchURL } from '@console/internal/actions/dashboards';
+import { RESULTS_TYPE } from '@console/internal/reducers/dashboards';
+import { RootState } from '@console/internal/redux';
+import * as classNames from 'classnames';
+import { sortable } from '@patternfly/react-table';
+import { fromNow } from '@console/internal/components/utils/datetime';
+import { Table, TableRow, TableData } from '@console/internal/components/factory';
+import { Timestamp } from '@console/internal/components/utils';
+import { StatusItem } from '@console/shared/src/components/dashboard/status-card/AlertItem';
+import { BlueInfoCircleIcon } from '@console/shared/src/components/status';
+import { getName, getNamespace } from '@console/shared/src/selectors/common';
+import {
+  VIRTUAL_MACHINE_IS_NOT_RUNNING,
+  NO_GUEST_AGENT_MESSAGE,
+} from '../../constants/vm/constants';
+import { VMStatus } from '../../constants/vm/vm-status';
+import { isGuestAgentInstalled } from '../dashboards-page/vm-dashboard/vm-alerts';
+import { VMStatusBundle } from '../../statuses/vm/types';
+import { VMIKind } from '../../types';
+
+const guestAgentURL = (name: string, namespace: string) =>
+  `apis/subresources.kubevirt.io/v1alpha3/namespaces/${namespace}/virtualmachineinstances/${name}/guestosinfo`;
```

Comment:
There's already logic for creating subresources url such as https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/src/selectors/vmi/combined.ts#L60

you should try to reuse that

Reply:
I think it's fixed now :-)